juego.py

```diff
@@ -77,11 +78,11 @@ def rand_generator_y():
 
         self.preguntas = pygame.sprite.Group()
         self.correct_number = number(rand_generator_x(), rand_generator_y(),
-                   fuente.render(
-                       self.resultado, True, (random.randint(100, 255),
-                                              random.randint(100, 255),
-                                              random.randint(100, 255))),
-                   True)
+                                     fuente.render(
+            self.resultado, True, (random.randint(100, 255),
+                                   random.randint(100, 255),
+                                   random.randint(100, 255))),
+            True)
```

Comment:
This isn't right as L81 is part of the call to `number` hence should be indented just four spaces and then the L82-L84 should be indented about 8 spaces from the `number` call and four from `fuente.render`